packages/gatsby-cli/src/reporter/index.js

```diff
@@ -139,6 +142,16 @@ const reporter: Reporter = {
         })
       },
       end() {
+        const diff = process.hrtime(startTime)
+
+        const NS_PER_SEC = 1e9
+        const MS_PER_NSEC = 1e6
+
+        trackCli(`ACTIVITY_DURATION`, {
+          name: name,
+          duration: Math.round((diff[0] * NS_PER_SEC + diff[1]) / MS_PER_NSEC),
```

Comment:
Could we use https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/src/util/calc-elapsed-time.js (or https://www.npmjs.com/package/convert-hrtime if precission returned by our util seems insufficient) instead of magic using magic numbers?

Reply:
yess, thanks!

Updated the pr to use the convert-hrtime module.